metricflow/engine/metricflow_engine.py

```diff
@@ -376,6 +388,7 @@ def __init__(
     @log_call(module_name=__name__, telemetry_reporter=_telemetry_reporter)
     def query(self, mf_request: MetricFlowQueryRequest) -> MetricFlowQueryResult:  # noqa: D
         logger.info(f"Starting query request:\n{indent(mf_pformat(mf_request))}")
+        logger.info(f"Setting ID generation to start at: {MetricFlowEngine._QUERY_ID_START_VALUE}")
```

Comment:
There's no reset call anywhere that I can find. Either remove the logging (and the constant, which is otherwise unused) or add the reset, whichever is appropriate.